api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -328,6 +329,13 @@ public User findUserByEmployeeId(final String employeeId, final boolean includeD
         return this.userDao.findByEmployeeId(employeeId, includeDeactivated);
     }
 
+    @Beta
```

Comment:
this is just to mark the method as a `Beta` feature. The signature may change if more complex queries are required